config-templates/authsources.php

```diff
@@ -28,26 +28,6 @@
         // Can be NULL/unset, in which case a builtin discovery service will be used.
         'discoURL' => null,
 
-        /*
-         * WARNING: SHA-1 is disallowed starting January the 1st, 2014.
-         *
-         * Uncomment the following option to start using SHA-256 for your signatures.
-         * Currently, SimpleSAMLphp defaults to SHA-1, which has been deprecated since
-         * 2011, and will be disallowed by NIST as of 2014. Please refer to the following
-         * document for more information:
-         *
-         * http://csrc.nist.gov/publications/nistpubs/800-131A/sp800-131A.pdf
-         *
-         * If you are uncertain about identity providers supporting SHA-256 or other
-         * algorithms of the SHA-2 family, you can configure it individually in the
-         * IdP-remote metadata set for those that support it. Once you are certain that
-         * all your configured IdPs support SHA-2, you can safely remove the configuration
-         * options in the IdP-remote metadata set and uncomment the following option.
-         *
-         * Please refer to the hosted SP configuration reference for more information.
-          */
-        //'signature.algorithm' => 'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256',
```

Comment:
Perhaps it's a good idea to leave this line in place

Reply:
What do you mean precisely?

Reply:
But this is the global config. Would really not want to encourage to switch back to sha1 globally. But instead add it only to specific entities that need it?

Reply:
Removed it here but added it to `saml20-sp-remote` as the most specific place to enable it.

Reply:
Okay, my idea was to make upgrading as painless as possible for deployers that need sha1 by giving them an example... On second thought however, they're likely to copy their existing config instead of using the template, so it may not make a whole lot of sense... Proper upgrade notes should suffice.

Reply:
Yes but I'd prefer to give them an example that enables it not globally. Because that might be the quickest fix but it's not "painless" in the sense that the pain is then that SHA-1 proliferates.

Can you add a review?